hcn/hcnsupport.go

```diff
@@ -76,6 +106,18 @@ func GetSupportedFeatures() SupportedFeatures {
 	features.NetworkACL = isFeatureSupported(globals.Version, NetworkACLPolicyVersion)
 	features.NestedIpSet = isFeatureSupported(globals.Version, NestedIpSetVersion)
 
+	// Only print the HNS version and features supported once, instead of everytime this is invoked. These logs are useful to
+	// debug incidents where there's confusion on if a feature is supported on the host machine. The sync.Once helps to avoid redundant
+	// spam of these anytime a check needs to be made for if an HNS feature is supported. This is a common occurrence in kubeproxy
+	// for example.
+	versionOnce.Do(func() {
+		logrus.WithFields(logrus.Fields{
+			"Major": globals.Version.Major,
+			"Minor": globals.Version.Minor,
+		}).Infof("HNS version")
```

Comment:
Also think this should probably not be `Info` level. Can we make it `Debug` or `Trace` instead?

Reply:
I don't believe k8s uses logrus so they don't set the level/expose configuring the logrus level by virtue of that. This would likely mean these logs won't show up

Reply:
Do we believe it's important for this log to show up in k8s output?

Reply:
@JocelynBerrendonner Had mentioned this would help with investigations regarding when a customer is questioning why certain HCN feature(s) aren't working/supported.

Reply:
If it's helpful to have for that case, then I think it's okay to leave it at `Info`, given it only logs once.